Updates to Campaign and Canvas Interactions Data for Data Retention doc (#7137)
braze-inc/braze-docs: _docs/_api/data_retention.md

#### Campaign and Canvas Interactions Data 

> Messaging interaction data refers to how a user interacts with a campaign or Canvas they received (for example, when a user opens campaign A or a user receives variant A). This data is used for retargeting. You can read more about messaging interaction data availability on [About messaging interaction data availability]({{site.baseurl}}/messaging_interaction_data/).

Reviewer: ```suggestion
Messaging interaction data refers to how a user interacts with a campaign or Canvas they received (for example, when a user opens campaign A or a user receives variant A). This data is used for retargeting. For more on the availability of message interaction data, refer to [About messaging interaction data availability]({{site.baseurl}}/messaging_interaction_data/).
```
